CHANGELOG.md

```diff
@@ -98,8 +98,8 @@ and this project adheres to
   `::addr_canonicalize`/`::addr_humanize` for consistency.
 - cosmwasm-vm: Add `BackendApi::addr_validate` to avoid having to do two calls
   from Rust into Go.
-- cosmwasm-vm: Upgrade Wasmer to 4.2.5; Bump `MODULE_SERIALIZATION_VERSION` to
-  "v9". ([#1992])
+- cosmwasm-vm: Upgrade Wasmer to 4.2.6; Bump `MODULE_SERIALIZATION_VERSION` to
+  "v9". ([#1992], [#2042])
```

Comment:
This change is good, but under the `## [2.0.0-rc.1] - 2023-02-09` header. Do we remove the pre-release headers when we get the full release (I perfer so)?

Reply:
Yes, that's what I did in the past. I think it makes the most sense since people who upgrade are probably most interested in the difference between 1.5 and 2.0.